src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -70,14 +70,38 @@ const QUERY = gql`
             url
           }
         }
+        product_generating_contexts {
+          product {
+            external_references {
+              accession_id
+              name
+              description
+              url
+              source {
+                id
+                name
+                url
```

Comment:
Is there any reason for us to request source url (here for protein, and above for transcript and for gene)? Are we using it anywhere?

Reply:
No, we don't. I've removed it.